src/features/presentation/components/webinaire/webinaire.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="row row-cols-1 row-cols-md-3 g-4 justify-content-center">
   <div class="col" *ngFor="let webinaire of webinairePresentation">
     <div class="card h-100 rounded-0">
-      <img src="/assets/img/presentation/webinaire/{{ webinaire.image }}" class="w-100" alt="" />
+      <img src="/assets/img/presentation/webinaire/{{ webinaire.image }}" class="w-100" alt="webinaire" />
```

Comment:
```suggestion
      <img src="/assets/img/presentation/webinaire/{{ webinaire.image }}" class="w-100" alt="" />
```
C'est une image d'illustration, je ne pense pas que la lecture à haute voie de `webinaire` par un lecteur d'écran apporte une information importante à un public qui a besoin d'un tel outil pour naviguer sur le site.